### PR TITLE
Richaudience Bid Adapter : add compability with GPID

### PR DESCRIPTION
**Pull request by richaudience**
## Type of change

- [ ] Bugfix
- [x ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

**Comment by richaudience** [comment]
@ChrisHuie 
@ikhemissi

File: modules/richaudienceBidAdapter.js [hunk 1]
```diff
@@ -54,7 +54,8 @@ export const spec = {
         scr_rsl: raiGetResolution(),
         cpuc: (typeof window.navigator != 'undefined' ? window.navigator.hardwareConcurrency : null),
         kws: (!isEmpty(bid.params.keywords) ? bid.params.keywords : null),
-        schain: bid.schain
+        schain: bid.schain,
+        gpid: raiSetPbAdSlot(bid)
```
Comment (lksharma) [inline modules/richaudienceBidAdapter.js:58]: Please update unit test to account for new `gpid` field.

Comment (richaudience) [reply modules/richaudienceBidAdapter.js:58]: i have added the test for gpid, please you can validate this?

Comment (richaudience) [reply modules/richaudienceBidAdapter.js:58]: @lksharma

**Review by lksharma — APPROVED** [review]
LGTM

**Pull request merged**